pkgs/by-name/by/byedpi/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
```

Comment:
I think our standard style would be as follows:

```
{ lib
, stdenv
, fetchFromGitHub
, nix-update-script
}:
```

Reply:
https://github.com/NixOS/rfcs/pull/166

Reply:
Btw I used `nixfmt-rfc-style` to format code

Reply:
Oh, I was not aware of this change, thank you for bringing it to my attention! I stand corrected then.